### PR TITLE
fix input checkbox and radio state

### PR DESCRIPTION
**Pull request by swaathee**
send values for radio and checkbox input type - this is being used in visualization for the checked state of the fields.

### CODE REVIEW

File: packages/clarity-js/src/core/scrub.ts [hunk 1]
```diff
@@ -9,8 +9,12 @@ let digitRegex = null;
 let letterRegex = null;
 let currencyRegex = null;
 
-export function text(value: string, hint: string, privacy: Privacy, mangle: boolean = false): string {
+export function text(value: string, hint: string, privacy: Privacy, mangle: boolean = false, type?: string): string {
     if (value) {
+        if (hint == "input" && type == "checkbox" || type == "radio") {
```
Comment (AbdelrhmanMagdy) [inline packages/clarity-js/src/core/scrub.ts:14]: Nit: `hint == "input" && (type == "checkbox" || type == "radio")`

**Review by AbdelrhmanMagdy — APPROVED** [review]

**Review by ender336 — APPROVED** [review]
:shipit:

**Review by keyrun — APPROVED** [review]
:shipit:

File: packages/clarity-js/src/core/scrub.ts [hunk 2]
```diff
@@ -9,8 +9,12 @@ let digitRegex = null;
 let letterRegex = null;
 let currencyRegex = null;
 
-export function text(value: string, hint: string, privacy: Privacy, mangle: boolean = false): string {
+export function text(value: string, hint: string, privacy: Privacy, mangle: boolean = false, type?: string): string {
     if (value) {
+        if (hint == "input" && (type == "checkbox" || type == "radio")) {
```
Comment (keyrun) [inline packages/clarity-js/src/core/scrub.ts:14]: >type == "checkbox" || type == "radio" [](http://example.com/codeflow?start=32&length=37)

Nit: type === "checkbox" || type === "radio"

**Pull request merged**